importer/tests/test_views.py

```diff
@@ -20,17 +23,113 @@
 class TestViews(LoreTestCase):
     """Hit each view."""
 
+    import_url = "/repositories/{0}/import/"
+    repository_url = "/repositories/{0}/"
```

Comment:
@carsongee: Based on our previous conversation about me nit-picking this, do you like these hanging out here in the class namespace instead of a `setUp` override, since it's less code? If so, I'll ignore it.